bodhi/tests/server/tasks/test_composer.py

```diff
@@ -2656,10 +2642,8 @@ def test_primary_arches_undefined(self):
 
         url = t._get_master_repomd_url('aarch64')
 
-        self.assertEqual(
-            url,
+        assert url ==\
```

Comment:
Please add a space between `==` and the backslash (see PEP8: "Always surround these binary operators with a single space on either side..."), here and in similar code below. Sorry for not spotting this earlier.

Reply:
Why flake8 didn't catch this?

Reply:
This should be fixed now, I found a few more in the code so I fixed them too.

Reply:
I think flake8 didn't catch this because of the backslash line-continuation.